app/bundles/ReportBundle/Controller/ReportController.php

```diff
@@ -783,14 +783,43 @@ public function exportAction($objectId, $format = 'csv')
         $fromDate = $session->get('mautic.report.date.from', (new \DateTime('-30 days'))->format('Y-m-d'));
         $toDate   = $session->get('mautic.report.date.to', (new \DateTime())->format('Y-m-d'));
 
-        $dynamicFilters = $session->get('mautic.report.'.$objectId.'.filters', []);
+        $date = (new DateTimeHelper())->toLocalString();
+        $name = str_replace(' ', '_', $date) . '_' . InputHelper::alphanum($entity->getName(), false, '-');
+        $options = array( 'dateFrom' => new \DateTime($fromDate),'dateTo' => new \DateTime($toDate));
 
-        $reportData = $model->getReportData($entity, null, [
-            'dateFrom'       => new \DateTime($fromDate),
-            'dateTo'         => new \DateTime($toDate),
-            'dynamicFilters' => $dynamicFilters,
-        ]);
+        if($format === 'html') {
+            if ($format === 'xlsx') {
```

Comment:
These two conditions doesn't make much sense. If $format === 'html', it can never === 'xlsx'